compiler/main/driver.cpp

```diff
@@ -98,6 +98,7 @@ static char moduleSearchPath[FILENAME_MAX] = "";
 static bool fBaseline = false;
 
 bool fLibraryCompile = false;
+bool fLibraryFortran = false;
```

Comment:
If we could keep the warnings under control, I think it'd be cool if `chpl --library` just automatically generated C, Fortran, and Python interfaces by default.

Reply:
I added a TODO: comment about that here.